crates/symbolicator/src/endpoints/mod.rs

```diff
@@ -1,28 +1,43 @@
-use actix_web::http::StatusCode;
-use actix_web::{App, HttpResponse, ResponseError};
+use axum::handler::{get, post};
+use axum::Router;
+use sentry::integrations::tower::NewSentryLayer;
 
-use crate::services::symbolication::MaxRequestsError;
+use crate::metrics::MetricsLayer;
 use crate::services::Service;
 
 mod applecrashreport;
-mod healthcheck;
+mod error;
 mod minidump;
 mod proxy;
 mod requests;
 mod symbolicate;
 
-/// Adds all endpoint routes to the app.
-pub fn configure(app: App<Service>) -> App<Service> {
-    app.configure(applecrashreport::configure)
-        .configure(healthcheck::configure)
-        .configure(minidump::configure)
-        .configure(proxy::configure)
-        .configure(requests::configure)
-        .configure(symbolicate::configure)
+pub use error::ResponseError;
+
+use applecrashreport::handle_apple_crash_report_request as applecrashreport;
+use minidump::handle_minidump_request as minidump;
+use proxy::proxy_symstore_request as proxy;
+use requests::poll_request as requests;
+use symbolicate::symbolicate_frames as symbolicate;
```

Comment:
nitpicking: Isn't this section above the mod section normally?  Also the `pub use` one should be the bottom of this section afaik https://www.notion.so/sentry/HOWTO-Code-Rust-at-Sentry-7b35f165b10b4492bb95ebe1471a9ada#57304f11240944338fe3828059bcf463